packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/Group.tsx

```diff
@@ -88,14 +89,18 @@ const useGroupEntryStyles = makeStyles<Theme, GroupStyleProps>((theme) => ({
   verticalSections: {
     display: 'flex',
     flexDirection: 'column',
-    alignItems: 'stretch',
     paddingLeft: theme.spacing(1),
     overflowX: 'auto',
+    gap: ({ borderStyle }) => (borderStyle ? borderStyle?.gap : null),
+    alignItems: ({ borderStyle }) => borderStyle?.alignItems || 'stretch',
```

Comment:
Those default values here are an issue. If I decide to add a border of "1px solid black", then my elements are aligned in a different manner. It does not make much sense. The default value should probably be removed on the frontend and instead specified on the backend.

Reply:
Customization is for flexbox only and the properties are not link to border anymore